plugins/parallel-tooltip.js

```diff
@@ -0,0 +1,162 @@
+(function (factory) {
+    if (typeof define === 'function' && define.amd) {
+        define(['tauCharts'], function (tauPlugins) {
+            return factory(tauPlugins);
+        });
+    } else if (typeof module === 'object' && module.exports) {
+        var tauPlugins = require('tauCharts');
+        module.exports = factory(tauPlugins);
+    } else {
+        factory(this.tauCharts);
+    }
+})(function (tauCharts) {
+
+    var _ = tauCharts.api._;
+
+    function ChartParallelTooltip(xSettings) {
+
+        var settings = _.defaults(
+            xSettings || {},
+            {
+                // add default settings here
+            });
+
+        var plugin = {
+
+            init: function (chart) {
+
+                this._cursor = null;
+                this._chart = chart;
+                this._tooltip = chart.addBalloon(
+                    {
+                        spacing: 3,
+                        auto: true,
+                        effectClass: 'fade'
+                    });
+
+                this._tooltip
+                    .getElement()
+                    .insertAdjacentHTML('afterbegin', this.template);
+
+                this._tooltip
```

Comment:
In my honest opinion we should create method for delegate events and expose to api